### ISSUE TITLE
kamailio master@4b4dc89 SIGSEGV ALERT: <core> [main.c:737]: handle_sigs(): child process 16507 exited by a signal 11

### ISSUE THREAD

**Issue by amessina** [kamailio/kamailio#393]
```
Using host libthread_db library "/lib64/libthread_db.so.1".
Core was generated by `/usr/sbin/kamailio -P /run/kamailio/kamailio.pid -m 128 -M 8'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x000000000041b5c9 in _dns_hash_remove (e=0x7f3321d5dde0) at dns_cache.c:495
495             debug_lu_lst("_dns_hash_remove: post rm:", &e->last_used_lst);
(gdb) bt full
#0  0x000000000041b5c9 in _dns_hash_remove (e=0x7f3321d5dde0) at dns_cache.c:495
        __func__ = "_dns_hash_remove"
#1  0x0000000000420eb8 in dns_cache_clean (no=4294967295, expired_only=1) at dns_cache.c:637
        e = 0x7f3321d5dde0
        now = 1894768549
        n = 0
        deleted = 0
        l = 0x7f3321d5ddf0
        tmp = 0x7f3321e6e528
#2  0x000000000041906d in dns_timer (ticks=1894768549, tl=0x7f331eaf10e8, data=0x0) at dns_cache.c:206
No locals.
#3  0x000000000049e6aa in slow_timer_main () at timer.c:1130
        n = 12
        ret = 4294967295
        tl = 0x7f331eaf10e8
        i = 243
        __func__ = "slow_timer_main"
#4  0x00000000005251dc in main_loop () at main.c:1645
        i = 2
        pid = 0
        si = 0x0
        si_desc = "udp receiver child=1 sock=[2001:55C:DC00:100:0:0:0:3]:5060\000\000\001\000\000\000h\227\036\060\063\177\000\000 \230\036\060\063\177\000\000\000n\\R\377\177\000\000\350\352\345\036\063\177\000\000\260~\260&3\177\000\000P\\O33\177\000\000 Q\260&3\177\000\000\003\000\000\000\001\000\000"
        nrprocs = 2
        woneinit = 1
        __func__ = "main_loop"
#5  0x000000000052b540 in main (argc=7, argv=0x7fff525c7158) at main.c:2591
        cfg_stream = 0x1279010
        c = -1
        r = 0
        tmp = 0x7fff525c7f3d ""
        tmp_len = 0
        port = 1700966438
        proto = 32767
        options = 0x74c478 ":f:cm:M:dVIhEeb:l:L:n:vKrRDTN:W:w:t:u:g:P:G:SQ:O:a:A:x:X:"
        ret = -1
        seed = 3889260924
        rfd = 4
        debug_save = 0
        debug_flag = 0
        dont_fork_cnt = 0
        n_lst = 0x0
        p = 0x7 <error: Cannot access memory at address 0x7>
        st = {st_dev = 20, st_ino = 1223079, st_nlink = 2, st_mode = 16872, st_uid = 986, st_gid = 983, __pad0 = 0, st_rdev = 0, st_size = 40, st_blksize = 4096, st_blocks = 0, st_atim = {tv_sec = 1447110524, tv_nsec = 573100727}, 
          st_mtim = {tv_sec = 1447110524, tv_nsec = 573100727}, st_ctim = {tv_sec = 1447110524, tv_nsec = 573100727}, __glibc_reserved = {0, 0, 0}}
        __func__ = "main"
```

**Comment by miconda** [comment]
This was master branch, right? Should be fixed by commit 70699ae8b9159c18bdd73c3092ccb19691edf0fd.

**Comment by amessina** [comment]
Yes, master branch. I will hopefully be able to test tonight or tomorrow.
